packages/mermaid/src/diagrams/class/classTypes.ts

```diff
@@ -106,7 +106,7 @@ export class ClassMember {
         this.visibility = firstChar as Visibility;
       }
 
-      if (lastChar.match(/[*?]/)) {
+      if (lastChar.match(/[$*]/)) {
```

Comment:
I'm not familiar with the specifics of class diagram syntax. Based on the docs, this change looks good.
But, doesn't this change break diagrams that had `?` previously? @knsv thoughts?

<img width="739" alt="image" src="https://github.com/mermaid-js/mermaid/assets/10703445/d8ffc24e-f25c-4c63-8eaf-8f0edaa2ff57">

Reply:
Thanks! Idk if the `?` was done by accident. I use mermaid class diagrams since 2022 and never only used `*` and `$`.

Reply:
I agree this will be a problem for people using the question mark. However, that has been a "hidden feature" in the docs, we have been consistent since 2019 * and $. My interpretation is a mistake, and I think we can go ahead.

We could allow question marks as well, but let's keep it in sync with the docs as the first option. In the very unlikely event that this results in outrage, we can add it.

Reply:
It looks like it was accidentally added in https://github.com/mermaid-js/mermaid/pull/4534, as it looks like `*` and `$` are the only two values that do anything in https://github.com/mermaid-js/mermaid/blob/05c2a11e3d265d99c21a1a190be75048c1c0e985/packages/mermaid/src/diagrams/class/classTypes.ts#L122-L131, so `/[$*]/` is correct!

Since the `?` doesn't seem to do anything currently, I believe this to be a safe change, even for people already using `?`.